readme.md

```diff
@@ -17,22 +31,20 @@ The basic usage is very straightforward: make your class extend `Entity`, and us
 import { Entity, EntityBuilder } from '@jobilla/entity';
 
 class User extends Entity {
-    // We instantiate with null to ensure the property exists
-    // at the time of hydration.
-    public name: string = null;
-    public email: string = null;
+    public name!: string;
```

Comment:
What will these now default to? Also should this behaviour be elaborated more upon in the readme?

Reply:
Default will _technically_ be `undefined`, but TS will show type errors if the raw data passed to `EntityBuilder.build*` don't have a non-null non-undefined string `name`. If `undefined` is an expected and acceptable type, it should be explicit in the type as `name: string | undefined`.

I just noticed that I already had the relevant documentation in the other currently open PR. I can move them here and close that one.